Dockerfile

```diff
@@ -1,8 +1,11 @@
 # Version values referenced from https://hub.docker.com/_/microsoft-dotnet-aspnet
 
-FROM mcr.microsoft.com/dotnet/sdk:8.0-cbl-mariner2.0. AS build
+FROM mcr.microsoft.com/dotnet/sdk:6.0-cbl-mariner2.0. AS build
+
+
 WORKDIR /src
-FROM mcr.microsoft.com/dotnet/aspnet:8.0-cbl-mariner2.0 AS runtime
+COPY [".", "./"]
+RUN dotnet build "./src/Service/Azure.DataApiBuilder.Service.csproj" -c Docker -o /out -r linux-x64
```

Comment:
are we bringing this -r `linux-x64` back? so rebuilding it intentionally as opposed to copying from the built artifacts?

Reply:
I didn't make changes to the Dockerfile... I don't see that in the changeset listed by GitHub either:
![image](https://github.com/Azure/data-api-builder/assets/6414189/d1f3b262-da26-4b78-9705-906ad49a5632)

Reply:
this was merging from main. updated in #2182 . This is the Dockerfile that had been historically present in our repo and was restored from adopting any OB related changes.

Reply:
The dockerfile in this repo does not contribute to the binaries built and released. Separate internal dockerfile is used.